hotham/src/rendering/camera.rs

```diff
@@ -6,41 +6,41 @@ use crate::util::posef_to_isometry;
 #[derive(Debug, Clone)]
 /// The Camera, or View, in a scene.
 pub struct Camera {
```

Comment:
It's still sometimes helpful to know the position of the camera; I'd leave the `position` variable here.

Reply:
The position variable was actually a transform with a bad name. I just gave it a better name.

Reply:
It wasn't *technically* a transform since the camera has no scale. :P But in any case, it would be good to still know where the camera is located, in world space!

Reply:
A transform without scale is still a transform.